pkg/master/master.go

```diff
@@ -124,6 +125,9 @@ type Config struct {
 	// Used to start and monitor tunneling
 	Tunneler genericapiserver.Tunneler
 
+	// Enables using master leases
+	EnableLeases bool
```

Comment:
+1 to @smarterclayton's statements, that was my ideal want and intent w/ compromise for review's sake.